### PR TITLE
Lf calibration

### PR DESCRIPTION
**Pull request by jmjefferson**
KiDS' removes the additive systematic (mean shear), however it should be done separately for North and South fields. In addition, this unifies calibrations for KiDS' and also makes an option to convert psf units from pixels to arcseconds. The metacal calibrations in specifically the psf diagnostics file also had a similar treatment as KiDS which was odd, so that was unified as well.

### CODE REVIEW

File: txpipe/utils/calibration_tools.py [hunk 1]
```diff
@@ -752,13 +738,14 @@ def __init__(
 
     def selector(self, data, i):
         x = data[self.x_name]
+        if ((self.shear_catalog_type=='lensfit') & (self.psf_conv == True) & ("T" in self.x_name)):
+            x = x * 0.214**2
```
Comment (yomori) [inline txpipe/utils/calibration_tools.py:742]: Again should probably make this an option in the config file

File: txpipe/utils/calibrators.py [hunk 2]
```diff
@@ -367,8 +370,14 @@ def apply(self, g1, g2, subtract_mean=True):
         """
 
         if subtract_mean:
-            g1 = (g1 - self.c[0]) / (1 + self.K)
-            g2 = (g2 - self.c[1]) / (1 + self.K)
+            Nmask = dec > -25.0
```
Comment (yomori) [inline txpipe/utils/calibrators.py:373]: Perhaps we can make this split dec an option as well? Then if someone for some reason wants to revert to analyzing everything at once they can just set this dec cut to like -90 or something.

**Review by yomori — CHANGES_REQUESTED** [review]
various changes

File: txpipe/test/test_cal_lensfit.py [hunk 3]
```diff
@@ -83,6 +92,7 @@ def test_mean_shear():
     # of the bins and the g1, g2 some fixed simple values, with metacal
     # factors perfectly applied
     data = {
+        "dec": np.random.normal(-5, 4, size=8),
```
[the remark this answers is not in the document]
Comment (jmjefferson) [reply txpipe/test/test_cal_lensfit.py:95]: nah it runs this test data thru the mean shear code, so for lensift it needs the dec to do the field splitting

[the remark this answers is not in the document]
Comment (jmjefferson) [reply txpipe/test/test_cal_lensfit.py:95]: so I just fed in the approximate dec bounds for the north field

**Comment by jmjefferson** [comment]
the galaxy star shear stage works for me using the actual lensfit data--I'm not sure why it isn't working in the check

**Review by elvinpoole — CHANGES_REQUESTED** [review]
Looks good to me. My only comment is we should probably make `dec_cut` a config option in the source selector stage. just to make it easy for us if we want to switch it off for a given run

**Review by yomori — APPROVED** [review]

**Review by elvinpoole — APPROVED** [review]

**Pull request merged**